dist/bash_completion/rkt.bash

```diff
@@ -1,860 +0,0 @@
-#!/bin/bash
```

Comment:
And I remember manually editing this file when removing the `--insecure-skip-verify` flag, stupid me.